senza/cli.py

```diff
@@ -292,119 +263,90 @@ def __init__(self, **kwargs):
     >>> test.Region
     'blubber'
-    >>> test.blubber
-    Traceback (most recent call last):
-      File "<stdin>", line 1, in <module>
-    AttributeError: 'AccountArguments' object has no attribute 'blubber'
-    '''
-    def __init__(self, region, **kwargs):
-        setattr(self, '__Region', region)
-        for key, val in kwargs.items():
-            setattr(self, '__' + key, val)
+    """
+    def __init__(self, region):
+        self.Region = region
+        self.__AccountAlias = None
+        self.__AccountID = None
+        self.__Domain = None
+        self.__MintBucket = None
+        self.__TeamID = None
+        self.__VpcID = None
 
     @property
     def AccountID(self):
-        attr = getattr(self, '__AccountID', None)
-        if attr is None:
-            accountid = get_account_id()
-            setattr(self, '__AccountID', accountid)
-            return accountid
-        return attr
+        if self.__AccountID is None:
+            self.__AccountID = get_account_id()
+        return self.__AccountID
 
     @property
     def AccountAlias(self):
-        attr = getattr(self, '__AccountAlias', None)
-        if attr is None:
-            accountalias = get_account_alias()
-            setattr(self, '__AccountAlias', accountalias)
-            return accountalias
-        return attr
-
-    @property
-    def Region(self):
-        return getattr(self, '__Region', None)
+        if self.__AccountAlias is None:
+            self.__AccountAlias = get_account_alias()
+        return self.__AccountAlias
 
     @property
     def Domain(self):
-        attr = getattr(self, '__Domain', None)
-        if attr is None:
-            return self.__setDomain()
-        return attr.rstrip('.')
-
-    def __setDomain(self, domainname=None):
-        domainlist = get_zone(domainname, all=True)
-        if len(domainlist) == 0:
+        if self.__Domain is None:
+            self.__setDomain()
+        return self.__Domain
+
+    def __setDomain(self, domain_name=None):
+        domain_list = get_zone(domain_name, all=True)
+        if len(domain_list) == 0:
             raise AttributeError('No Domain configured')
-        elif len(domainlist) > 1:
+        elif len(domain_list) > 1:
             domain = choice('Please select the domain',
-                            sorted(domain['Name'].rstrip('.') for domain in domainlist))
+                            sorted(domain['Name'].rstrip('.')
+                                   for domain in domain_list))
         else:
-            domain = domainlist[0]['Name'].rstrip('.')
-        setattr(self, '__Domain', domain)
+            domain = domain_list[0]['Name'].rstrip('.')
+        self.__Domain = domain
         return domain
 
-    def splitDomain(self, domainname):
-        self.__setDomain(domainname)
-        if domainname.endswith('.{}'.format(self.Domain)):
-            return domainname[:-len('.{}'.format(self.Domain))], self.Domain
+    def splitDomain(self, domain_name):
```

Comment:
This function name has to be camelCase? I think camelCase is only "necessary" for the amazon stuff.